tools/analysis_tools/analyze_results.py

```diff
@@ -220,7 +342,8 @@ def main():
     outputs = mmcv.load(args.prediction_path)
 
     result_visualizer = ResultVisualizer(args.show, args.wait_time,
```

Comment:
Please also verify that the effect of `demo/create_result_gif.py` is correct when `overlay-gt-pred=False`

Reply:
I will do it in the next pr.